backend/src/graphql/resolver/util/findContributionMessages.ts

```diff
@@ -0,0 +1,36 @@
+import { In } from '@dbTools/typeorm'
+import { ContributionMessage as DbContributionMessage } from '@entity/ContributionMessage'
+
+import { ContributionMessageType } from '@enum/ContributionMessageType'
+import { Order } from '@enum/Order'
+
+interface FindContributionMessagesOptions {
+  contributionId: number
+  pageSize: number
+  currentPage: number
+  order: Order
+  showModeratorType?: boolean
+}
+
+export const findContributionMessages = async (
+  options: FindContributionMessagesOptions,
+): Promise<[DbContributionMessage[], number]> => {
+  const { contributionId, pageSize, currentPage, order, showModeratorType } = options
+
+  const messageTypes = [ContributionMessageType.DIALOG, ContributionMessageType.HISTORY]
+
+  if (showModeratorType) messageTypes.push(ContributionMessageType.MODERATOR)
```

Comment:
I would have passed the whole array into the function, so that the resolver defines all types he wants to see.

Reply:
Can we do this, when there is a requirement for it?